plugins/backend/backstage-plugin-argo-cd-backend/README.md

```diff
@@ -16,12 +16,16 @@ argocd:
       instances:
         - name: argoInstance1
           url: https://argoInstance1.com
-          token: ${ARGOCD_AUTH_TOKEN}
+          token: ${ARGOCD_AUTH_TOKEN} # optional 
         - name: argoInstance2
           url: https://argoInstance2.com
+          # dedicated username/password for this instance
+          username: ${ARGOCD_USERNAME_INSTANCE_2} # optional 
+          password: ${ARGOCD_PASSWORD_INSTANCE_2} # optional 
 ```
 
 Add the required auth tokens to environmental variables, `ARGOCD_USERNAME` and `ARGOCD_PASSWORD`.
+Define a `username` and a `password` for each instance if needed.
```

Comment:
I think the explanation of the variables and settings could be a little clearer here. But not a deal breaker per se.

I am assuming I have three options:

token, explicit config based user name and password, or providing a username and password via environment variables. Is my assumption correct?

Reply:
Yes you right @punkle. I updated the doc with a better structure and the clear "options" . What do you think ?